pkg/kubelet/apis/config/types.go

```diff
@@ -389,6 +389,11 @@ type KubeletConfiguration struct {
 	// +featureGate=GracefulNodeShutdown
 	// +optional
 	ShutdownGracePeriodCriticalPods metav1.Duration
+	// PodPriorityShutdownGracePeriods specifies the shutdown grace period for Pods based priorities
+	// +featureGate=GracefulNodeShutdown
```

Comment:
Is this controlled by both feature gates or just the new one?

Reply:
This only takes effect when both feature gates are enabled

Reply:
Got it, let's leave a comment than here that `PodPriorityBasedGracefulShutdown` requires `GracefulNodeShutdown` feature gate to also be enabled

Reply:
It has been added to the next line, but it happened to be separated by this comment 😂

https://github.com/kubernetes/kubernetes/pull/102915/files#diff-8a6475c76df7985730ad410e5d1a6caccbbe3ba44853ed773722063cf3625639R393-R394

Reply:
ah, I was referring to comment

```
// PodPriorityShutdownGracePeriods specifies the shutdown grace period for Pods based priorities
```
and add something like

```
// PodPriorityShutdownGracePeriods specifies the shutdown grace period for Pods based priorities. Requires the GracefulNodeShutdown feature gate to be enabled.
```